### ISSUE TITLE
Accessibility: MAS40B: Inspect: Localized Control Type Property is wrong for the "Ext Install Python" read only text

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#55]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/items?itemName=donjayamanne.python in EDGE
2. Navigate to the "Ext Install Python" read only text available under "Installation" 
3. Observe the "Localized Control Type" property in Inspect

**Actual Result:**
The Localized Control Type" property is "Edit"
**_Narrator Behaviour:_**
Narrator announces as "Read Only Ext Install Python"

**Expected Result:**
The Localized Control Type property should be "Text" instead of "Edit"

**User Impact:**
The localized Control Type property should be correct as because the command is non editable.

**Screenshot:**
![localized control type is edit instead of text](https://cloud.githubusercontent.com/assets/24825441/21758804/2238398e-d665-11e6-9d73-10b6ea2d54ad.png)

**Comment by waderyan** [comment]
Marketplace issue.
